packages/angular-output-target/README.md

```diff
@@ -40,12 +41,12 @@ export const config: Config = {
-| `customElementsDir`           | This is the directory where the custom elements are imported from when using the [Custom Elements Bundle](https://stenciljs.com/docs/custom-elements). Defaults to the `components` directory. Only applies when `includeImportCustomElements` is `true`.                                                                        |
+| Property               | Description                                                                                                                                                                                                                                                                             |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `componentCorePackage` | The NPM package name of your Stencil component library. This package is used as a dependency for your Angular wrappers.                                                                                                                                                                 |
+| `directivesProxyFile`  | The output file of all the component wrappers generated by the output target. This file path should point to a location within your Angular library/project.                                                                                                                            |
+| `directivesArrayFile`  | The output file of a constant of all the generated component wrapper classes. Used for easily declaring and exporting the generated components from an `NgModule`. This file path should point to a location within your Angular library/project.                                       |
+| `valueAccessorConfigs` | The configuration object for how individual web components behave with Angular control value accessors.                                                                                                                                                                                 |
+| `excludeComponents`    | An array of tag names to exclude from generating component wrappers for. This is helpful when have a custom framework implementation of a specific component or need to extend the base component wrapper behavior.                                                                     |
+| `outputType`           | The type of output to generate. If `component`, the output target will generate lazy loaded component wrappers tied to an Angular module. If `scam`, the output target will generate a single component angular module for each component. If `standalone`, standalone component wrappers will be generated. |
```

Comment:
>  The type of output to generate. If `component`, the output target will generate lazy loaded component

This isn't necessarily true since you can have the following, which would using dist-custom-elements with a single angular module.

```js
angularOutputTarget({
  componentCorePackage: '@ionic/core',
  directivesProxyFile: '../angular/src/lazy-loaded/directives/proxies.ts',
  customElementsDir: 'components',
  // You can add outputType: 'component', but since `component` is the default it's not required.
});
```

Reply:
Addressed here: https://github.com/ionic-team/stencil-ds-output-targets/pull/365/commits/62baec20cf7544388fcbeb5becbe4f514d0e204e

Reply:
The changes look good. Should we add stronger verbiage around customElementsDir being required for scam/standalone?

Reply:
Good thought, it really isn't "recommended", it is required. I'll adjust after stand-up with some stronger language around the requirements.

Reply:
Here's another attempt: https://github.com/ionic-team/stencil-ds-output-targets/pull/365/commits/1e987c2fb01937e6ebd78eb201095056c51ce4ea